.travis.yml

```diff
@@ -8,4 +8,4 @@ script:
   - npm install coveralls
   - npm test -- -u --coverage
 after_success:
-  - npm run coverage
+  - COVERALLS_REPO_TOKEN=olYi6pj3rhE6JWKlOdLq3siCyVWxddv2E npm run coveralls
```

Comment:
Why leave such sensitive information in a public repository? This should be included in the CI environment settings.

Reply:
Thank you @emmabaye, great attention to detail. I hope you also put up a ticket to remove it from https://github.com/openmrs/openmrs-ocl-client/blob/master/.coveralls.yml.

Reply:
Yes, it should be considered. I wasn't part of the team that starting the project. I didn't realise it was there. It's not right.

Reply:
Addressed